src/harness/security_testcase.py

```diff
@@ -140,6 +154,146 @@ def findIndexMatching(array, regex):
     self.assertTrue(finished_check_idx > 0)
     self.assertTrue(finished_check_idx > cipher_check_idx)
 
+
+
+  def assertVerifyServerCertificateSuceed(self, base_url, ciphers, client_cert, client_key):
+    """ Specific to SCS_20 testcase
```

Comment:
If this is really used only in SCS.20, then let's put the code in the same file. The purpose of this file is to contain helper functions that will be reused across two or more test cases.

Reply:
We   agree and the    "assertVerifyServerCertificateSuceed()" method will be moved   to "test_WINNF_FT_S_SCS_20()" function of   "WINNF_FT_S_SCS_testcase.py". This change will be submitted as part   of the new PR for SCS.20